templates/index.html

```diff
@@ -1,11 +1,21 @@
 <!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Weather</title>
-</head>
-<body>
-    <h1>Hello World</h1>
-</body>
```

Comment:
Can I why did you remove this code snippet?

Reply:
i want try out bulma template